R/slip-days.R

```diff
@@ -0,0 +1,102 @@
+#' Apply Slip Days
+#' 
+#' This function applies any slip day policies within the policy file. 
+#'
+#' @param gs A Gradescope dataframe
```

Comment:
In terms of argument naming, are we still calling this `gs` across the codebase (now that we're working with Canvas too?). Minimally, this should be changed to "A gradebook dataframe".

A more general discussion: we may want to class our "canonical" data frame with "nemo_df" or something.